config/jobs/kubernetes/test-infra/fejta-bot-periodics.yaml

```diff
@@ -1,5 +1,47 @@
 # this file should contain all periodic jobs that use the fejta-bot token
 periodics:
+- name: periodic-api-review-help
+  interval: 1h
+  decorate: true
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/commenter:v20170808-abf66782
+      command:
+      - /commenter
+      args:
+      # pull requests against master branches in the kubernetes org labeled kind/api-change
+      # exclude PRs that are in progress, held, or need rebase (typically aren't ready for API review).
+      # exclude PRs already labeled api-review or tracked in the API review project
+      # exclude PRs that already have the comment text
+      - |-
+        --query=org:kubernetes
+        is:pr
+        base:master
+        label:kind/api-change
+        -label:do-not-merge/work-in-progress
+        -label:do-not-merge/hold
+        -label:needs-rebase
+        -label:api-review
+        -project:kubernetes/13
+        NOT "complete the pre-review checklist and request an API review"
+      - --updated=5m
```

Comment:
@fejta Is this 5 minutes? Like, it will only find PRs updated in the last 5 minutes?

Reply:
Other way around: flag.DurationVar(&o.updated, "updated", 2*time.Hour, "Filter to issues unmodified for at least this long if set")

It excludes things updated in the last 5 minutes. Mostly just a sanity check to prevent hot loops.